src/renderers/dom/client/utils/createObjectWithProto.js

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
Just inline the IE8 path in `SyntheticEvent.augmentClass`.

No need for a fancy module and no need to check for Object.create if the IE8 path is equivalent anyway.